docs/tutorials/nb_imputation_comparison.ipynb

```diff
@@ -0,0 +1,271 @@
+    "\n",
+    "[![open in colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/ing-bank/probatus/blob/master/docs/tutorials/nb_imputation_comparison.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook explains how the `ImputationSelector` class works in `probatus`. With `ImputationSelector` you can compare multiple imputation strategies\n",
+    "and choose a strategy which works the best for a given model and a dataset.\n",
+    "Currently `ImputationSelector` supports any scikit learn compatible imputation strategy. For categorical variables the missing values are replaced by `missing` token and `OneHotEncoder` is applied. The user supplied imputation strategies are applied to numerical columns only. \n",
+    "Support for user supplied imputation strategies for categorical columns can be added in the future releases.\n",
+    "\n",
+    "Let us look at an example and start by importing all the required classes and methods.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "%load_ext autoreload\n",
+    "%autoreload 2\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "pd.set_option('display.max_columns', 100)\n",
+    "pd.set_option('display.max_row', 500)\n",
+    "pd.set_option('display.max_colwidth', 200)\n",
+    "from probatus.missing_values.imputation import ImputationSelector\n",
+    "from probatus.utils.missing_helpers import generate_MCAR\n",
+    "import lightgbm as lgb \n",
+    "from sklearn.linear_model import LogisticRegression\n",
+    "from sklearn.experimental import enable_iterative_imputer  \n",
+    "from sklearn.impute import KNNImputer,SimpleImputer,IterativeImputer\n",
+    "from sklearn.datasets import make_classification"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us create a classification dataset to apply the various imputation strategies.\n",
+    "\n",
+    "Add missing values to the dataset. `generate_MCAR` method randomly adds missing values to the dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Shape of X,y : (2000, 20),(2000,)\n"
+     ]
+    }
+   ],
+   "source": [
+    "n_features = 20\n",
+    "X,y = make_classification(n_samples=2000,n_features=n_features,random_state=123,class_sep=0.3)\n",
+    "X = pd.DataFrame(X, columns=[\"f_\"+str(i) for i in range(0,n_features)])\n",
+    "print(f\"Shape of X,y : {X.shape},{y.shape}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "execute_result",
+     "data": {
+      "text/plain": [
```

Comment:
Is it possible to make this print horizontal not vertical?

Reply:
Done.